web/components/manage-vault.tsx

```diff
@@ -968,7 +968,7 @@ export const ManageVault = ({ match }) => {
                 <div className="mt-2">
                   <p className="text-sm text-gray-500">
                     Choose how much USDA you want to burn. Burning will include a stability fee of{' '}
-                    {stabilityFee / 1000000} USDA, so take this into account.
+                    <span className="font-semibold">{stabilityFee / 1000000} USDA</span>, so take this into account.
```

Comment:
We can remove "so take this into account" as our frontend logic is now taking it into account.

Reply:
Yeah. Good catch. I'll update it.